locale/ar_IQ/locale.xml

```diff
@@ -55,12 +55,12 @@
 	<message key="submission.license.cc.by-nd4">CC Attribution-NoDerivatives 4.0</message>
 	<message key="submission.license.cc.by4">CC Attribution 4.0</message>
 	<message key="submission.license.cc.by-sa4">CC Attribution-ShareAlike 4.0</message>
-	<message key="submission.license.cc.by-nc-nd4.footer"><![CDATA[<a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" /></a><br />هذا العمل مرخص حسب <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>.]]></message>
```

Comment:
Currently creativecommons.org seems to be automatically redirecting http traffic to https, which I suppose works in favor of this broader change.  The change still feels too broad to me.  When you use the [Choose a License tool](https://creativecommons.org/choose/), they currently output code with the `<a href>` as http, and the `<img src>` as https.  I think we should mimic that model.